README.md

```diff
@@ -154,7 +154,13 @@ liminal build
 
 You'll see that a number of outputs indicating various docker images built.
 
-3. Deploy the pipeline:
+3. Create a kubernetes local volume
```

Comment:
In case your Yaml includes working with volumes (link: https://github.com/apache/incubator-liminal/blob/6253f8b2c9dc244af032979ec6d462dc3e07e170/docs/getting_started.md#mounted-volumes)

please first run the following command: